Sources/AppcuesKit/Presentation/ExperienceRendering/LifecycleEvent.swift

```diff
@@ -46,15 +46,15 @@ internal enum LifecycleEvent: String, CaseIterable {
         }
 
         if let stepIndex = stepIndex, let step = experience.step(at: stepIndex) {
-            properties["stepId"] = step.id.uuidString.lowercased()
+            properties["stepId"] = step.id.appcuesFormatted
             properties["stepType"] = step.type
             // stepIndex is added primarily for use by the debugger
             properties["stepIndex"] = stepIndex.description
         }
 
         if let error = error {
             properties["message"] = error.message
-            properties["errorId"] = error.id.uuidString
+            properties["errorId"] = error.id.appcuesFormatted
```

Comment:
This one was not `lowercased()` before, but I'm assuming it should be.

Reply:
This one doesn't matter since its generated on the client, but it doesn't hurt to consistent with what gets set to the backend.